build/utils/cli.js

```diff
@@ -121,10 +146,11 @@ const getImageReferences = (source_directories, ignored_files = []) => flatten(f
             if (!line) {
                 return;
             }
-            const matched = line.match(/Images\/app_2\/([^';]*)|<img([^>]*)>|background: url([^-color][^;]*)/g);
+            const matched = line.match(/Images\/app_2\/([^';]*)|<img([^>]*)>|background[-image]+: url([^-color][^;]*)/g);
```

Comment:
@easteregg Would you plz consider the regex again? e.g should `[-image]+` be `(-image)?` or it'd be better to add more groups to make clear the `or`s

Reply:
`(-image)?` means zero or one of `-image` so it doesn't miss `background: url(`.

Reply:
the `?` is belong to the suggestion and not for the question :-P